### DISCUSSION TITLE
Asynchronic draft phase

### DISCUSSION THREAD

**Discussion by K-Hartmann** [bafolts/terraforming-mars#283] [Ideas]
Currently the draft phase is implemented strictly parralel. After i pick out of 4 cards, I can not see the next three cards until everyone has made their first pick.
If instead I could see the next three cards as soon as I picked my first card and the player before me made his pick, the draft would be smother and quicker. This way, if one player is afk, the draft stacks would "build up" infront of that player and he could make all 4 picks one after another.

**Comment by vincentneko** [comment]
I'm afraid this could be very difficult to put in place in regards on how the game is working.
I suggest we leave this here for the moment as there are many other game areas that require most immediate attention.
